config.yaml.tpl

```diff
@@ -39,7 +35,7 @@ ${dcos_exhibitor_storage_backend == "azure" ? dcos_exhibitor_azure_prefix== "" ?
 ${dcos_master_external_loadbalancer == "" ? "" : "master_external_loadbalancer: ${dcos_master_external_loadbalancer}"}
 ${dcos_master_discovery == "master_http_loadbalancer" ? dcos_num_masters == "" ? "" : "num_masters: ${dcos_num_masters}" : ""}
 ${dcos_master_discovery == "master_http_loadbalancer" ? dcos_exhibitor_address== "" ? "" : "exhibitor_address: ${dcos_exhibitor_address}" : ""}
-${dcos_master_discovery == "static" ? dcos_master_list== "" ? "" : "master_list: ${dcos_master_list}" : ""}
+${dcos_master_discovery == "static" ? dcos_master_list== "" ? "" : "master_list: \n  ${indent(2,dcos_master_list)}" : ""}
```

Comment:
the template resource can only receive strings. https://github.com/dcos-terraform/terraform-template-dcos-core/pull/6/files#diff-7a370d8342e7203b805911c92454f0f4R80

Reply:
Got it. Nice touch.

Reply:
We will need to probably update all the top level modules to covert this to a list as well to avoid terraform type issues, but this change makes sense.

Reply:
thanks. Yeah i did PRs on the other modules.

However I still expect issues in some cases here. Reason is that object structured variables also need indentation. 

But we should come up with test scenarios and do fixes based on those.